app/assets/javascripts/discourse/app/templates/components/uppy-backup-uploader.hbs

```diff
@@ -0,0 +1,4 @@
+<label class="btn" disabled={{uploading}} title={{i18n "admin.backups.upload.title"}}>
+  {{d-icon "upload"}}&nbsp;{{uploadButtonText}}
```

Comment:
I think it would be better to have a specific class for that label like `backup-uploaded` and format it with CSS instead of space character

Reply:
I just removed the space from both templates, no class is needed because d-icon already has a margin